From adf32284bb5f9c5815f2c1461cf6437568081adf Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Wed, 12 May 2021 18:23:28 +0700
Subject: [PATCH 175/180] arm64: dts: renesas: r9a07g044l2-smarc: Add Mic
 routing

This patch adds audio routing for Mic with Bias to reduce noise
when doing audio capture.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts b/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
index cf62aba..4d0cee5 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
@@ -115,6 +115,12 @@
 		simple-audio-card,bitclock-master = <&cpu_dai>;
 		simple-audio-card,frame-master = <&cpu_dai>;
 
+		simple-audio-card,widgets = "Microphone", "Microphone Jack";
+		simple-audio-card,routing =
+			    "L2", "Mic Bias",
+			    "R2", "Mic Bias",
+			    "Mic Bias", "Microphone Jack";
+
 		cpu_dai: simple-audio-card,cpu {
 			sound-dai = <&ssi0>;
 		};
-- 
2.7.4

